Scenes/MainMenu/SubMenus/Settings/Settings.gd

```diff
@@ -1,8 +1,31 @@
 extends Control
 
+onready var mute_box = get_node("CenterContainer/PanelContainer/MarginContainer/VBoxContainer/HBoxContainer4/VBoxContainer2/HBoxContainer/MuteCheckBox")
+onready var volume_slider = get_node("CenterContainer/PanelContainer/MarginContainer/VBoxContainer/HBoxContainer4/VBoxContainer2/VolumeSlider")
+onready var mute_label = get_node("CenterContainer/PanelContainer/MarginContainer/VBoxContainer/HBoxContainer4/VBoxContainer2/HBoxContainer/MuteLabel")
+
+func _ready():
+	if AudioServer.is_bus_mute(0):
+		mute_label.text = "yes"
+		mute_box.pressed = true
+		
+	volume_slider.value = AudioServer.get_bus_volume_db(0)
+
+
+func _on_VolumeSlider_value_changed(value):
+	#var percent_value = value / $MarginContainer/CenterContainer/VBoxContainer/VolumeSlider.max_value
+	#var new_text = str(int(round(percent_value * 100 + 100))) + "%"
```

Comment:
Unnecessary comments

Reply:
I didnt delete them because maybe in the future we will need to show % value of sound volume